lib/rdstation/error_handler.rb

```diff
@@ -13,6 +13,8 @@ def raise_error
       raise error_class, array_of_errors.first if error_class < RDStation::Error
 
       error_class.new(array_of_errors).raise_error
+    rescue JSON::ParserError => error
```

Comment:
Since we are handling this error directly in the API response, in which cases that exception would be raised?

Reply:
this will be raised when the body ain't a valid JSON. The most comon scenario for this to happen is when we have a downtime on the API.